plugins/modules/docker_plugin.py

```diff
@@ -3,7 +3,8 @@
 #
 # Copyright: (c) 2021 Red Hat | Ansible Sakar Mehra<@sakarmehra100@gmail.com | @sakar97>
 # Copyright: (c) 2019, Vladimir Porshkevich (@porshkevich) <neosonic@mail.ru>
```

Comment:
Same here.

Reply:
Fixed in b6721b6bb7d1e4136cef1218ef3d033899057aa9.